build-system/compile/compile.js

```diff
@@ -50,7 +50,7 @@ const MAX_PARALLEL_CLOSURE_INVOCATIONS =
  * production use. During development we intend to continue using
  * babel, as it has much faster incremental compilation.
  *
- * @param {string} entryModuleFilename
+ * @param {string[]} entryModuleFilename
```

Comment:
`compile` takes `string[]|string` - is that accurate or should it only take `string[]`?

also nit: make param plural `entryModuleFilenames`

Reply:
Array or string is correct. (It's a relic of how we do things differently during dep checking. I'm writing in a reactor of this file, so feel free to ignore the arg renaming for now.)

Reply:
`string[]|string` makes sense. There are no places where it is called with `string` so I guess I missed that it was an option.